tests/bluetooth/tester/src/gap.c

```diff
@@ -720,9 +795,19 @@ static void conn_param_update(const u8_t *data, u16_t len)
 		   status);
 }
 
+static void set_mitm(const u8_t *data, u16_t len)
+{
+	LOG_WRN("Use CONFIG_BT_SMP_ENFORCE_MITM instead");
```

Comment:
With this option we can set the MITM bit based on the security level, maybe we should switch the security level we call 'bt_conn_set_security' with based on this command?

At the moment we always call with `BT_SECURITY_L2`, we could switch to `BT_SECURITY_L3` when mitm is requisted by the test.

Reply:
That's a good idea,  but I wonder if this would have unexpected consequences elsewhere in other tests? not sure, just thinking out loud

Reply:
This is a new command, so if the default is L2, and using this command would change it to L3 then there shouldn't be any issues with the tests. If anything then`CONFIG_BT_SMP_ENFORCE_MITM=n` is the change I would question if had unexpected consequences for the tests.